### DISCUSSION TITLE
无法连接和速度慢的特殊情况

### DISCUSSION THREAD

**Discussion by Wodksk** [XIU2/CloudflareSpeedTest#523] [讨论]
### 问题描述

使用了新的ip和公开ip皆无法使用其他工具（也是ip优选）全都超时

### 软件版本

2.2.5

**Comment by XIU2** [comment]
什么意思？没看懂

**Comment by Wodksk** [comment]
ipv4全都无法使用，更新过，公开的也无法使用ipv6速度几乎为0

**Comment by XIU2** [comment]
Issues 只用来反馈 CloudflareST 软件自身问题，其他问题找我没用，我也无法解决，因此请使用 Discussions 讨论

**Comment by microka** [comment]
最近时不时出这种幺蛾子，太糟心了。

**Comment by Wodksk** [comment]
在https://www.v2ex.com/t/758955 中曾讨论过，现在此问题依然存在，在8点后网速便非常慢，出现-1延迟等，但是根据几次尝试ipv6并不受影响或较小，且不会出现无法连接，这个问题一直都存在，但是在使用阿里（唯一特例）的反代时速度相对较快，也许是gfw的主动防御
